kai_analyzer_rpc/main.go

```diff
@@ -25,30 +27,16 @@ func main() {
 	lspServerPath := flag.String("lspServerPath", "", "this will be the path to the lsp")
 	bundles := flag.String("bundles", "", "Comma separated list of path to java analyzer bundles")
 	depOpenSourceLabelsFile := flag.String("depOpenSourceLabelsFile", "", "Path to the dep open source labels file")
+	pipePath := flag.String("pipePath", "", "Path to the pipe to use for bi-directional communication")
+	logVerbosity := flag.Int("verbosity", -4, "how verbose would you like the logs to be, error logs are 8, warning logs are 4 info logs are 0 and debug logs are -4, going more negative gives more logs.")
```

Comment:
Just FYI, I have another PR which uses levels strings, so that we can expose it directly in vscode config. This is fine for now. I will rebase my PR